src/features/store/ui/CartList.tsx

```diff
@@ -0,0 +1,46 @@
+import * as S from './styles';
+import type Item from '@type/Item';
+import StoreItem from './StoreItem';
+const testItem: Item[] = [
+  {
+    id: 1,
+    name: '해적 베레모',
+    image: '해적-베레모.svg',
+    cost: 500,
+    category: 'accessories',
+  },
+  {
+    id: 2,
+    name: '해적 의상',
+    image: '해적-의상.svg',
+    cost: 1000,
+    category: 'clothes',
+  },
+  {
+    id: 3,
+    name: '해초의 습격',
+    image: '해초의-습격.svg',
+    cost: 500,
+    category: 'profile',
+  },
+  {
+    id: 4,
+    name: '해적 베레모',
+    image: '해적-베레모.svg',
+    cost: 500,
+    category: 'color',
+  },
+];
+
+export default function CartList() {
+  return (
+    <>
+      <S.StoreCartListWrapper>
+        {testItem.map(item => (
+          <StoreItem {...item} />
+        ))}
```

Comment:
```suggestion
        {testItem.map(item => (
          <StoreItem key={item.id} {...item} />
        ))}
```
`{...item}`으로하면 key Prop을 명시적으로 안넘겨도 됐었나요? 안넘겨도 된다면 key의 값은 뭐가 되는 걸까요?

Reply:
key가 StoreItem 내부에서 사용되고 있었습니다.. {...itme}으로 해도 key를 명시적으로 지정해줘야 경고메세지가 나지 않는 것 같습니다!

Reply:
@rhehfl @bluetree7878 https://github.com/modern-agile-team/8term-coko-Front/pull/84#discussion_r1874887436 이쪽 코멘트에 남겨놨습니다